src/server/app/admin/controllers/users/users.authentication.server.controller.js

```diff
@@ -83,7 +83,7 @@ function signupAlert(user, req, res) {
 		name: user.name,
 		username: user.username,
 		appName: config.app.title,
-		url: `${config.app.baseUrl}/#/admin/users`
```

Comment:
Not in love with doing it this way, but @mheir mentioned that adding this to the config is better than hardcoding it since it is an angular thing we currently have configured, but might not be forever.  Also this extraRoute is not working atm I will try and fix it now.